scripts/mkcloud

```diff
@@ -519,6 +523,23 @@ EOS
     echo "you can now proceed with installing crowbar"
 }
 
+function createadminsnapshot()
+{
+    virsh shutdown $cloud-admin
+    wait_for 150 1 "virsh domstate $cloud-admin|grep shut.off" 'admin node to shut down'
+    lvremove -f $admin_node_disk.snap
+    safely lvcreate -l100%ORIGIN -s -n $cloud.admin.snap $admin_node_disk
+    virsh start $cloud-admin
+    wait_for_crowbar_ssh
+}
+
+function restoreadminfromsnapshot()
+{
+    virsh destroy $cloud-admin
```

Comment:
this 2 steps will only be used manually for the developer's convenience. so in most of the cases the `createadminsnapshot` step has been done before. but yes a simple check for the existence of a `.snap` here doesn't hurt I agree.